images/win/Windows2016-Readme.md

```diff
@@ -68,30 +69,37 @@
 
 ### CLI Tools
 - Azure CLI 2.8.0
-- azcopy 10.4.3
+- azcopy 10.5.0
 - Azure DevOps CLI extension 0.18.0
-- AWS CLI 2.0.28
+- AWS CLI 2.0.30
 - AWS SAM CLI 0.53.0
-- Alibaba CLI 3.0.26
+- AWS Session Manager CLI 1.1.61.0
+- Alibaba Cloud CLI 3.0.26
 - Cloud Foundry CLI 6.51.0
 - Hub CLI 2.14.2
-- Google Cloud SDK 299.0.0
+- Google Cloud SDK 300.0.0
+
+### Rust packages:
+- bindgen 0.54.1
+- cbindgen 0.14.3
+- cargo-audit 0.12.0
+- cargo-outdated v0.9.10

```

Comment:
Rust packages section looks strange here because Rust tool is located significantly above.
May be something like that?
```
### Rust
Version: 1.44.1
#### Packages
- bindgen 0.54.1
- cbindgen 0.14.3
- cargo-audit 0.12.0
- cargo-outdated v0.9.10
```

cc: @miketimofeev @al-cheb @dmitry-shibanov